### ISSUE TITLE
Circular rebalance returns "payment failed" but actually succeeds

### ISSUE THREAD

**Issue by sendbitcoin** [Ride-The-Lightning/RTL#474]
Sometimes when I try to do a circular rebalance I will get "payment failed" but when I click try again and pick another channel and enter the same amount and try to do it again it tells me "invoice already paid" -> closing the circular rebalance dialog shows the same old channel balances -> only after a page refresh are the channel balances updated to reveal that the payment did indeed succeed.

Version of RTL: 0.8.4-beta
Version of lnd: LND 0.11.0-beta
Version of btcpayserver docker: BTCPayServer v1.0.5.5
Browser: Firefox

**Comment by ShahanaFarooqui** [comment]
I was unable to reproduce this senario with 20 different channels and amounts. Closing the ticket for now. Please feel free to reopen if it can be reproduced by following some steps.

**Comment by sendbitcoin** [comment]
@ShahanaFarooqui the problem regarding balance not updating immediatly and needing a page refresh happens all the damn time even on normal successful rebalances.
I just did 2 rebalances and on the second one I got the issue again. -> perhaps firefox issue? dunno what to say as far as reproducing goes as it is very easy to reproduce for me -> just do a rebalance and look at the total amount of local and remote balance (they wont add up to the full amount in the channel) -> refresh page and it updates to the correct amount

the Problem with "payment failed" but actually succeeds is more rare and happens randomly -> probably some weird edge case regarding timeout and how RTL judges if it's failed or succeeded? maybe rtl timeout lower then lnd timeout? -> don't have a reliable way to reproduce as it only happens occasionally... but it's a real thing that definitely happens
